### ISSUE TITLE
Treemap click event

### ISSUE THREAD

**Issue by tanvi-cota** [plotly/angular-plotly.js#105]
I have been trying to use Treemap with angular-plotly. It seems to render fine, but I have difficulty with the click event, in Treemap.
```
<plotly-plot *ngIf="plotlyGraph?.data && plotlyGraph?.data.length > 0" [useResizeHandler]="true"
    (plotly_click)="onSelect($event)" [data]="plotlyGraph.data"
    [layout]="plotlyGraph.layout" [config]="plotlyGraph.config"></plotly-plot>
```

I use the above generic html to create different visuals.  I am facing issue when trying to handle the click/select event on Treemap.
I read documentation in plotly.js and they have some event for treemap `plotly_treemapclick` (https://github.com/plotly/plotly.js/pull/4454). But this event doesn't seem to be available here.

**Comment by andrefarzat** [comment]
Hello @tanvi-cota , looks like these clicks (`plotly_sunburstclick` and `plotly_treemapclick`) are new on the `plotly.js`. 
Do you think we can simply add these events as properties on `plotly-plot` ? like: `(treemapclick)` and `(sunburstclick)`

**Comment by tanvi-cota** [comment]
I think we should be able to. I was going through other reported issues, and we had something similar to this request. And there we just used it as you suggested

**Comment by tanvi-cota** [comment]
@andrefarzat any idea when can this feature/update be available?

**Comment by andrefarzat** [comment]
@tanvi-cota adding events are really easy. Just published the `1.7.0` version. Please, update and check it out 😸

**Comment by tanvi-cota** [comment]
@andrefarzat It seems I cant take the latest and work on it, due to the error - #104
